### PR TITLE
Solved module 2 task 1

### PR DESCRIPTION
**Pull request by pbochniak**

### CODE REVIEW

File: module-2/task1/main.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+	"github.com/pbochniak/computator/fibonacci"
+)
+
+func main() {
+	var inputNumber string
+	var number interface{}
+	fmt.Printf("Give me a number: ")
+	fmt.Scanf("%s", &inputNumber)
+	if u, err := strconv.ParseUint(inputNumber, 10, 0); err == nil {
```
Comment (yushni) [inline module-2/task1/main.go]: In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.

https://golang.org/doc/effective_go#if

Comment (yushni) [reply module-2/task1/main.go]: Also, I think that better to use a separate `func` for the cast. 
My suggestions: 

```

func main() {
	var input string
	fmt.Printf("Give me a number: ")
	fmt.Scanf("%s", &input)

	number, err := castNumber(input)
	if err != nil {
		fmt.Println("failed to cast number ", err)
		return
	}

	fmt.Printf("Fibonacci sequence for given number (%v [%T]) equals %v\n", number, number, fibonacci.Fibonacci(number))
}
func castNumber(input string) (interface{}, error) {
	if u, err := strconv.ParseUint(input, 10, 0); err == nil {
		return uint(u), nil
	}
	if i, err := strconv.ParseInt(input, 10, 0); err == nil {
		return int(i), nil
	}
	if f, err := strconv.ParseFloat(input, 64); err == nil {
		return f, nil
	}
	return nil, fmt.Errorf("invalid type")
}

```

**Review by yushni — APPROVED** [review]

File: module-2/task1/main.go (hunk 1 above)
Comment (pbochniak) [reply module-2/task1/main.go]: "In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted." 

But which else was unnecessary in my case ? I know that that sometimes is better to skill **else** when return or break happends in if block (same as in python), but in my case when casting wasn't in separate function I couldn't avoid elses.

Comment (yushni) [reply module-2/task1/main.go]: That's why i suggested to use separate function.

**Pull request merged**